extensions/extension.go

```diff
@@ -193,8 +247,15 @@ func saveFunc(env any, _ string, args []object.Object) object.Object {
 }
 
 func loadFunc(env any, _ string, args []object.Object) object.Object {
-	file := args[0].(object.String).Value
-	// Open file for reading.
+	file := ".gr"
+	if len(args) != 0 {
+		file = args[0].(object.String).Value
+		var err error
+		file, err = sanitizeFileName(file)
+		if err != nil {
+			return object.Error{Value: err.Error()}
+		}
```

Comment:
As the code is duplicated between save and load, shouldn't it be moved into sanitizeFilename, by passing a []object.Object as argument

Reply:
fixed, thanks https://github.com/grol-io/grol/pull/130/files